docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '3.0'
+
+services:
+  grafana:
+    container_name: 'grafana-athena-datasource'
+    build:
+      context: ./.config
+      args:
+        grafana_version: ${GRAFANA_VERSION:-9.1.2}
+    ports:
+      - 3000:3000/tcp
+    volumes:
+      - ./dist:/var/lib/grafana/plugins/grafana-labs-amazon-athena-datasource
```

Comment:
This file is auto generated, no? From where did it get the `grafana-labs-amazon-athena-datasource` folder name?

Reply:
technically the name it got was `rafanaabs-mazonthena-datasource` lol, but I think it comes from https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/common/docker-compose.yaml . I'm not sure why it's different from the `container_name` above, because the template implies that they should be the same

Reply:
Ok weird! Let's change folder name to `grafana-athena-datasource` then.